src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGenerator.cs

```diff
@@ -29,25 +20,17 @@ public void Execute(GeneratorExecutionContext context)
 			//	Debugger.Launch();
 			//}
 
-			//
-			// Lock the current generator instance, as it may be invoked concurrently
-			// in the context of omnisharp when saving and editing fast enough, causing
-			// corruption issues in the GenerationRunInfoManager.
```

Comment:
The locking was added for something specific to GenerationRunInfoManager which has been removed.